src/ui/controls/logsdialog.hpp

```diff
@@ -19,10 +19,11 @@ namespace NickvisionTubeConverter::UI::Controls
     	 * @param destructiveText The text of the destructive button
     	 * @param suggestedText The text of the suggested button
     	 */
-		LongMessageDialog(GtkWindow* parent, const std::string& title, const std::string& description, const std::string& cancelText, const std::string& destructiveText = "", const std::string& suggestedText = "");
+		LogsDialog(GtkWindow* parent, const std::string& title, const std::string& description, const std::string& cancelText, const std::string& destructiveText = "", const std::string& suggestedText = "");
```

Comment:
Change `const std::string& description` to `const std::string& logs` here just like you did in the .cpp file.
Also update the documentation comment to reflect this change.